### DISCUSSION TITLE
Change color date less 2 days on Table panel

### DISCUSSION THREAD

**Discussion by sanchomuzax** [grafana/grafana#46924] [Dashboards, Panels & Visualizations]
It has a column with date type cell, and need to change the color of this date when it's less than 2 days (or other time range).
Example: Get the current date “2022-03-16” - 2 days “2022-03-14” any date less than or equal to 2022 -03-14 change the color to red, is it possible?

![image](https://user-images.githubusercontent.com/2911588/159919959-9e2f2772-18a8-4c86-8c43-ddc1b0d47043.png)

[Similar question...](https://community.grafana.com/t/change-color-date-less-2-days/62416)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
